frontend/src/views/InstanceDashboard.vue

```diff
@@ -95,7 +85,7 @@ export default defineComponent({
 
       if (!uiStateStore.getIntroStateByKey("guide.instance")) {
         setTimeout(() => {
-          state.showGuide = true;
+          event?.emit(EventType.EVENT_HELP, "instance");
```

Comment:
how do we differentiate between instance.detail and instance.list

Please take a look at the type in Issue / Activity to make it more structure, otherwise, it will be a hard to maintain all help guide

Reply:
I added a REAME.md under /public/help to specify file naming rules, not sure if this is what you asked for. 
Please take another look.

Reply:
I still doubt our DevMar or avid contributor could develop and integrate the content on their own. Let's improve this in the followup PR.

Reply:
Okay, I'll work on it more to improve its accessibility.